distribution/docs/src/main/resources/_contents/_draft-solr-contents/managing-solr-contents.adoc

```diff
@@ -1,39 +1,382 @@
+
+==== Installing
+
+===== Prerequisites
+
+Before the ${ddf-solr} Application can be installed:
+
+* the ${branding} Kernel must be running,
+* the ${branding} Platform Application must be installed, and
+* the ${branding} Solr Application must be installed
+
+===== Installing
+
+. Edit the <INSTALLATION_DIR>/etc/system.properties
+
+.. Comment out the Solr Client Configuration for *Cloud Solr Client* and *Http Solr Client* sections.
+.. Uncomment the section for the *Embedded Solr Client*:
+
+.system.properties
+----
+solr.client = EmbeddedSolrServer
+----
 
-.${ddf-solr} Optional Configurations
-[cols="1,3m,3" options="header"]
+==== Configuring Embedded Solr Server
+
+No configuration is necessary for the embedded Solr Server.
+The standard installation described above is sufficient.
+
+However, this component _can_ be configured to specify the directory to use for data storage.
+
+The configurable properties for the SCP are accessed from the *Embedded Solr Provider* configurations in the Web Console.
+
+===== Configurable Properties
+
+[cols="1,1,1,4a,1,1" options="header"]
 |===
-|Name
-|Configuration ID
+
+|Title
+|Property
+|Type
 |Description
+|Default Value
+|Required
+
+|Data Directory File Path
+|`dataDirectoryPath`
+|String
+|Specifies the directory to use for data storage. The server must be shutdown for this property to take effect. If a filepath is provided with directories that don't exist, SCP will attempt to create those directories. Out of the box (without configuration), the SCP writes to `<DISTRIBUTION_INSTALLATION_DIRECTORY>/data/solr`.
 
-include::{adoc-include}/_tables/conf-ddf.catalog.source.solr.SolrCatalogProvider-table-contents.adoc[]
+If `dataDirectoryPath` is left blank (empty string), it will default to `<DISTRIBUTION_INSTALLATION_DIRECTORY>/data/solr`.
+
+If data directory file path is a relative string, the SCP will write the data files starting at the installation directory. For instance, if the string `scp/solr_data` is provided, the data directory will be at `<DISTRIBUTION_INSTALLATION_DIRECTORY>/scp/solr_data`.
+
+If data directory file path is `/solr_data` in Windows, the Solr Provider will write the data files starting at the beginning of the drive, e.g., `C:\solr_data`.
+
+It is recommended that an absolute filepath be used to minimize confusion, e.g., `/opt/solr_data` in Linux or `C:\solr_data` in Windows. Permissions are necessary to write to the directory.
+|
+|No
+
+|Force Auto Commit
+|`forceAutoCommit`
+|Boolean / Checkbox
+|
+[IMPORTANT]
+====
+*Performance Impact* +
+Only in special cases should auto-commit be forced. Forcing auto-commit makes the search results visible immediately.
+====
+|
+|No
 
 |===
+
+==== Move Solr Data to a New Location
+
+If SCP has been installed for the first time, changing the Data Directory File Path property and restarting the distribution is all that is necessary because no data had been written into Solr previously.
```

Comment:
will spell out to Solr Provider (there is more to remove Catalog from Solr in the docs)